### DISCUSSION TITLE
31

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#746] [Comments]
http://jojoldu.tistory.com/31

**Comment by jojoldu** [comment]
작성자: Jack

Can't read from server. It may not have the appropriate access-control-origin settings. 설치후 이런 오류메세지가 나오는데 어떻게 해결해야 할까요?

**Comment by jojoldu** [comment]
작성자: 창천향로

그 문제는 제가 봤을때 CORS 문제인것 같습니다^^; 적용한 프로젝트와 호출한 URL의 도메인이 같은지 확인이 먼저 필요할것 같습니다
비슷한 내용을 깃헙 이슈에 등록되있는데 참고하시면 좋을것 같습니다!
https://github.com/swagger-api/swagger-ui/issues/146

감사합니다^^

**Comment by seokWonYoon** [comment]
안녕하세요.
swagger로 api문서만들기를 따라하였는데 
현재 구성이 controller / model(vo) / persistance(dao겸 mapper) / service 로 되어있습니다. 

저같은 경우엔 jpa와 h2를 쓰지 않고 vo클래스를 만들어서 mybatis로 mysql을 활용하여 사용중입니다. 
그래서 컨트롤러에 repository를 오토와이어 할 것이 없어서 하지 않았습니다. 

다른 부분은 잘 따라해서 괜찮게 나오는데 모델 스키마 부분이 나오지 않습니다. 이 부분이 레포지터리를 오토와이어 하지 않아서 나오는 문제일까요?

**Comment by wyjax** [comment]
좋은 글 감사합니다.

**Comment by dongseoki** [comment]
글 보고 사이드 프로젝트에 적용해 보았습니다. 감사합니다~!

**Comment by underdarks** [comment]
감사합니다 !  API 개발 문서 정리하려고 찾다가 좋은 글을 찾게 되었습니다 바로 한번 적용해보겟습니다 ㅎㅎ

**Comment by wyjax** [comment]
thank you for your written
